docs/hub/repositories-getting-started.md

```diff
@@ -0,0 +1,224 @@
+---
+title: Getting Started with Repositories
+---
+
+<h1>Getting Started with Repositories</h1>
+
+This beginner-friendly guide will help you get the basic skills you need to create and manage your repository on the Hub. Each section builds on the previous one, so feel free to choose where to start!
+
+### Requirements
```

Comment:
I think we should follow the heading hierarchy and use `h2`'s for all these sections instead of skipping to a `h3`.

Reply:
Ah I totally missed that, good catch!